spec/index.bs

```diff
@@ -315,13 +315,12 @@ like {{GPUBuffer}} or {{GPUQueue}}, have shared state which can be simultaneousl
 This allows race conditions to occur, similar to those of accessing a `SharedArrayBuffer`
 from multiple Web Workers, which makes the thread scheduling observable.
 
-The set of mitigations to address this attack vector, employed by user agents,
-can be a subset of [mitigations](https://hacks.mozilla.org/2020/07/safely-reviving-shared-memory/)
-against the malicious `SharedArrayBuffer` use. In particular,
-the user agent may limit sharing any object handles (such as {{GPUBuffer}}) to [=agents=] belonging
-to the same [=browsing context group=], and only if the
-[cross-origin isolated](https://web.dev/coop-coep/) policies are in place. 
-It may also serialize Web Workers that share any handles to prevent any concurrency entirely.
+WebGPU addresses this by only only exposing the shared-mutable-state operations to
```

Comment:
Does this mean we disallow Serializing GPUBuffer etc., or does it mean when you serialize a GPUBuffer you get a lesser version of GPUBuffer that can't map?

Reply:
The intent is to have it as a latter, although I've yet to see how this will work out. A PR to change the actual IDL will follow.

Reply:
I like the idea and I think it will work.

Reply:
I added the relevant `[CrossOriginIsolated]` now to enforce this.